src/gui_common/tooltip/ToolTipHelper.cs

```diff
@@ -45,4 +45,13 @@ public static DefaultToolTip CreateDefaultToolTip()
 
         callbackData.Add(toolTipCallbackData);
     }
+
+    /// <summary>
+    ///   Registers a Control mouse enter/exit event to display a custom tooltip from the given tooltip and group name.
+    /// </summary>
+    public static void RegisterToolTipForControl(this Control control, string tooltipName, string groupName,
+        List<ToolTipCallbackData> callbackDatas)
```

Comment:
```suggestion
        List<ToolTipCallbackData> callbackData)
```
Data is already plural (AFAIK) so you can't really add "s" to it.